tests/run-pass/clock.rs

```diff
@@ -0,0 +1,7 @@
+// compile-flags: -Zmiri-disable-isolation
+
+use std::time::SystemTime;
+
+fn main() {
+   let _now = SystemTime::now();
```

Comment:
You implemented two syscalls but test only one thing... why that?

Reply:
I should add a comment for that, one is the linux shim and the other is the macos one.